src/common/services/session/session.service.js

```diff
@@ -135,7 +135,7 @@ function session( $cookies, $rootScope, $http, $timeout, envService ) {
           resetKey: resetKey
         }
       } ) )
-      .map( ( response ) => response.data );
+      .mergeMap( () => signIn(email, password) );
```

Comment:
Since we have all the data needed when the password is reset, why not just sign the user in immediately? After the reset modal is closed which performs a page refresh, the user should be good to go. If this isn't desired, this could be removed and we could rely on the page refresh and then the session enforcer for sign in.